src/app/screens/Keysend/index.tsx

```diff
@@ -103,21 +80,31 @@ function Keysend(props: Props) {
             <>
               {destination && <PublisherCard title={destination} />}
 
-              <dl className="shadow bg-white dark:bg-surface-02dp pt-4 px-4 rounded-lg my-6 overflow-hidden">
-                {elements().map(([t, d], i) => (
-                  <Fragment key={`element-${i}`}>
-                    <dt className="text-sm font-semibold text-gray-500">{t}</dt>
-                    <dd className="text-sm mb-4 dark:text-white break-all">
-                      {d}
-                    </dd>
-                  </Fragment>
-                ))}
+              <dl className="text-sm shadow bg-white dark:bg-surface-02dp pt-4 px-4 rounded-lg my-6 overflow-hidden">
+                <dt className="font-medium text-gray-800 dark:text-white">
+                  {t("receiver.label")}
+                </dt>
+                <dd className="mb-4 dark:text-white break-all">
+                  {destination}
+                </dd>
+                <div className="font-semibold text-gray-500 mb-4">
```

Comment:
Ar, sorry, just saw this now.  
We can't use a `div` within a `dl`. `dl` can only have `dt/dd`.  
Please have a look how to make this semantic.